demo/clusterctl-examples/sample-kubeadm-ctlplane-crio-ubuntu.rc

```diff
@@ -0,0 +1,260 @@
+#
+# Pod and service CIDR for the cluster
+#
+export POD_CIDR=192.168.0.0/18
+export SERVICE_CIDR=10.96.0.0/12
+
+#
+# Cluster API host and port for *workload cluster* to be provisioned.
+# This is *not* the host/IP and port of the management cluster.
+# This is typically the virtual IP for an HA workload cluster.
+#
+# NOTE: make sure to pick an host/IP that is *unique* for each
+# cluster and make sure the host/IP is outside the ranges
+# of any IP pool.
+#
+export CLUSTER_APIENDPOINT_HOST=192.168.75.98
```

Comment:
Does this mean my BareMetalHosts need fixed IP addresses for this to work? In my lab they get DHCP addresses from Ironic DHCP server.

Reply:
Yes, the samples follows upstream metal3-dev-env which creates a fix virtual IP, which managed by keepalived.